modular_skyrat/modules/cyborg/code/robot_upgrades.dm

```diff
@@ -244,3 +244,28 @@
 		var/obj/item/inducer/cyborg/inducer = locate() in target_robot.model
 		if (inducer)
 			target_robot.model.remove_module(inducer, TRUE)
+
+// Borg picket sign upgrade
+/obj/item/borg/upgrade/sign
+	name = "Cyborg Multi-purpose advertisement tool upgrade"
+	desc = "A Picket sign upgrade for cyborgs"
```

Comment:
"A sign unit for cyborgs, allowing them to easily advertise products through an intuitive, wave-able sign." 

Would probably fit a bit better

Reply:
This sounds better.

Reply:
your's sounds better. My bad.